pkgs/development/libraries/libtermkey/default.nix

```diff
@@ -14,7 +14,7 @@ stdenv.mkDerivation rec {
     ++ stdenv.lib.optional stdenv.isDarwin "LIBTOOL=${libtool}/bin/libtool";
 
   nativeBuildInputs = [ pkgconfig ];
-  buildInputs = [ libtool ncurses ];
+  buildInputs = [ libtool ncurses unibilium ];
```

Comment:
`libtool` should probably be moved to `nativeBuildInputs`

Reply:
done